src/model/user.rs

```diff
@@ -190,13 +190,7 @@ impl CurrentUser {
     pub async fn guilds(&self, http: impl AsRef<Http>) -> Result<Vec<GuildInfo>> {
         let mut guilds = Vec::new();
         loop {
-            let mut pagination = http
-                .as_ref()
-                .get_guilds(
-                    &GuildPagination::After(guilds.last().map_or(GuildId(1), |g: &GuildInfo| g.id)),
-                    100,
-                )
-                .await?;
+            let mut pagination = http.as_ref().get_guilds(None, Some(100)).await?;
```

Comment:
This has to be reverted back to the original code. The current version will only fetch at most 100 guilds and skip the rest if the bot is in more than 100 guilds (it may also enter an infinite loop if `Http::get_guilds` returns exactly 100 guilds).
```suggestion
            let mut pagination = http
                .as_ref()
                .get_guilds(
                    Some(&GuildPagination::After(guilds.last().map_or(GuildId(1), |g: &GuildInfo| g.id))),
                    Some(100),
                )
                .await?;
```

Reply:
Oh I see, thanks. I'll change it back